### DISCUSSION TITLE
Please tell me your MBTI（性格タイプ）and your hobbies if you like❤️

### DISCUSSION THREAD

**Discussion by anzhitingg** [NAIST-SE/AttractingContributors#1165] [2024 Attraction Contributions]
The MBTI is based on the influential theory of psychological types proposed by Swiss psychiatrist Carl Jung in 1921, which was partially based on the four elements of classical cosmology. Jung speculated that people experience the world using four principal psychological functions—sensation, intuition, feeling, and thinking—and that one of these four functions is dominant in an individual, a majority of the time.
In MBTI theory, the four categories are introversion/extraversion, sensing/intuition, thinking/feeling, and judging/perceiving. According to the MBTI, each person is said to have one preferred quality from each category, producing 16 unique types.
Want to know what type you are? Find out if you're interested!
You can choose a type that best matches you from here:
https://www.16personalities.com/ja/%E6%80%A7%E6%A0%BC%E3%82%BF%E3%82%A4%E3%83%97
Or click here to take the test：
https://www.16personalities.com/
Please edit the ReadMe file and send a pull request.
https://github.com/anzhitingg/AttractingContributors-MBTI.git
Thank you very much！！！

**Comment by 3207-Rhims** [comment]
I made a PR on you, would you be kind to contribute in my project ? https://github.com/3207-Rhims/AttractingContributors-Movie

**Reply by anzhitingg** [reply]
I've done it!
Thank you very much😊

**Comment by ryosuke-sota** [comment]
I just made a PR on your repo. </br>Could you contribute to mine? </br>Here : [my project link (fav drinks)](https://github.com/ryosuke-sota/AttractingContributors.git)  </br>Thank you.

**Reply by anzhitingg** [reply]
I've done it!
Thank you very much🍷
